core/object.h

```diff
@@ -335,6 +335,11 @@ protected:
 			m_inherits::_notificationv(p_notification, p_reversed);                                                                     \
 	}                                                                                                                                   \
                                                                                                                                         \
+protected:                                                                                                                              \
+	virtual void _destruct() {                                                                                                \
```

Comment:
I'm running clang-format manually with a hotkey on VS. Maybe I forgot this file. I'm fixing it.